portfolio/src/main/webapp/script.js

```diff
@@ -28,39 +32,44 @@ function addRandomFact() {
   factContainer.innerText = fact;
 }
 
-function getName(){
-  console.log('Fetching name!');
+/*************************
+ *     Greeting
+ ************************/
+function greetingMaker() {
+  const greetings = ["Bonjour!","Ahoj!","Hola!","Hello!","Guten Tag!", "Yasou!"];
 
-  // The fetch() function returns a Promise because the request is asynchronous.
-  const responsePromise = fetch('/data');
+  // Pick a random greeting.
+  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-  // When the request is complete, pass the response into handleResponse().
-  responsePromise.then(handleResponse);
+  // Add it to the page.
+  const greetingContainer = document.getElementById('greeting-container');
+  greetingContainer.innerText = greeting;
 }
 
 
-/**
- * Handles response by converting it to text and passing the result to
- * addQuoteToDom().
- */
-function handleResponse(response) {
-  console.log('Handling the response.');
+/*************************
+ *   COMMENTS SECTION 
+ ************************/
 
-  // response.text() returns a Promise, because the response is a stream of
-  // content and not a simple variable.
-  const textPromise = response.text();
+ function comments(){
```

Comment:
I would also remove these unused function stubs for now for the same reason as the UpvoteServlet.

Reply:
done